src/root/port.c

```diff
@@ -335,12 +335,26 @@ int Port::stricmp(const char *s1, const char *s2)
 
 float Port::strtof(const char *p, char **endp)
 {
-    return static_cast<float>(::strtod(p, endp));
+    if(endp)
+        return static_cast<float>(::strtod(p, endp)); // does not set errno for underflows, but unused
+
+    _CRT_FLOAT flt;
+    int res = _atoflt(&flt, p);
```

Comment:
Well this is great and all, but VS2010 errors out because it has `_atoflt` declared as taking a `char*`.